rust/src/lib/nm/query_apply/ip.rs

```diff
@@ -183,6 +184,17 @@ fn nm_dhcp_duid_to_nmstate(nm_setting: &NmSettingIp) -> Option<Dhcpv6Duid> {
     }
 }
 
+pub(crate) fn nm_forwarding_to_nmstate(nm: &NmSettingIp) -> Option<bool> {
+    nm.forwarding.and_then(|val| match val {
+        0 => Some(false),
+        1 => Some(true),
```

Comment:
Please add some constant variables to the top of the file like FORWARDING_YES and use them here.

Reply:
This function should not exist. We should depend on nispor for querying kernel, not NM.

Reply:
But nm_ip_setting_to_nmstate4 exists. What is it used for?

Reply:
I removed this function, thanks!